docs/infrastructure/requesting-infrastructure.md

```diff
@@ -51,3 +51,19 @@ Please be aware that:
 - the Infrastructure Team will initially setup the resource in a secure manner
   and provide guidance on how to keep it that way, however it is your
   responsibility to ensure that the resource remains secure
+
+## On Call availability
+
+It is expected that during some events within the competition calendar there
+will be a need for members of the Infrastructure Team to be available to quickly
+respond to any infrastructure issues.
+
+See [On Call availability](./operations.md#on-call-availability) for more
+details of what this is.
+
+Please include:
```

Comment:
Whilst possibly a complete guess, it would be a good idea to have an understanding from the requester things which _could_ go wrong.

Reply:
Hrm, perhaps we should include something like "services/infrastructure that you will be depending on"?

Reply:
3e1a0e5